data/json/terrain.json

```diff
@@ -540,6 +568,7 @@
         "type" : "terrain",
         "id" : "t_grass",
         "name": "grass",
+        "description": "A beautiful, if unkempt section of hardy Kentucky blue grass.  Cutting the grass short enough would destroy the root system, causing this area to turn into a patch of dirt.",
```

Comment:
"if unkempt" is a parenthetical statement, and should have a closing comma as well. I would recommend using either parentheses or hyphens to more abruptly separate it, however.

> A beautiful - if unkempt - section of hardy Kentucky...

Also: "bluegrass" is one word

Reply:
Looks like I must've copied and pasted an older edit, as I could've sworn I changed these, as mentioned in the other comment.